DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>Is DP0.2 NB 22a really a thing? If it's an existing notebook, we should link to it. If it's not publicly available, then I wouldn't refer to it by name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Not yet, it is currently under development by Leanne (and I guess should be called 13b now that we updated the number of this one to 13a). I'm not sure that there is an estimated time for completion yet. I will try to clarify this.</p>

Reply:
<p>Noted instead that it is in preparation</p>